conda-recipe/meta.yaml

```diff
@@ -17,12 +17,14 @@ requirements:
     - numpy
     - six
     - matplotlib
+    - enaml
```

Comment:
The enaml lines need to be 

```
 - enaml [py2k]
```
so that we do not require enaml with python 3 (which does not exist).  See the DataProtal meta.yaml for an example.